src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java

```diff
@@ -138,16 +134,15 @@ public JsonNode getGivenSprintData(int sprintId, JsonNode allSprintData){
 
 
     @GetMapping("/{projectSlug}/getTotalPoints")
-    public List<TotalPoints> getMilestoneCompletedPoints(@PathVariable String projectSlug){
+    public List<TotalPoints> getMilestoneCompletedPoints(@PathVariable String projectSlug, int projectId){
         return milestoneService.getMilestoneTotalPoints(authenticationService.getAuthToken(), projectSlug);
-
     
     }
 
-    //Work Caapcity
-    @GetMapping("/{projectId}/getCompletedPoints")
-    public List<CompletedPoints> getMilestoneTotalCompletedPoints(@PathVariable int projectId){
-        return milestoneService.getMilestoneCompletedPoints(authenticationService.getAuthToken(), projectId);
+    //Work Capacity
+    @GetMapping("/{projectSlug}/getCompletedPoints")
+    public List<CompletedPoints> getMilestoneTotalCompletedPoints(@PathVariable String projectSlug){
```

Comment:
Add         int projectId = projectService.getProjectId(authenticationService.getAuthToken(), projectSlug);
here it will be helpful in writing test cases